### PR TITLE
Bug fix; custom image storage read

### PR DESCRIPTION
**Pull request by milismsft**
Fix unexpected exception thrown while attempting to retrieve the list of custom images from a specified storage account.

### CODE REVIEW

**Comment by milismsft** [comment]
@scotmoor @JargoonPard PTAL

File: clouddriver-web/config/clouddriver.yml [hunk 1]
```diff
@@ -104,15 +104,15 @@ azure:
...
-        region: "westus"
+  #    customImages:
+  #    - scs: ${AZURE_SCS1}
+  #      blobDir: "system/Microsoft.Compute/Images/vhds"
+  #      osType: "Windows"
+  #      region: "westus"
+  #    - scs: ${AZURE_SCS2}
+  #      blobDir: "vhds"
+  #      osType: "Linux"
+  #      region: "westus"
```
Comment (ghost) [inline clouddriver-web/config/clouddriver.yml:119]: Add an explanation for the commented out lines and some minimal instructions for their usage. Take a look at line 281 for an example of how they are currently doing that.

**Comment by ghost** [comment]
Tip: If you want to review the changes without the noise of the whitespace changes simply add ?w=1 to the end of the url.

**Comment by milismsft** [comment]
@scotmoor @JargoonPard PTAL

**Pull request closed without merging**